### PR TITLE
Restructure GECToR codebase and add conda recipes

### PR DESCRIPTION
**Pull request by Frost45**
In this PR, the GECToR code base is restructured to better support installing in a conda environment. `utils` and `data` have been moved into `gector`. As a result, imports and paths of files have been changed across code files and test files.

In addition to this, `data/vocabulary` has been removed as the vocabulary can now be found under `test_fixtures`.

All changes made to `train.py` have been made by `black` apart from the imports.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,30 +1,35 @@
...
     packages=find_packages(),
-    version='2.0',
-    license='Apache-2.0 License',
-    description='GECToR – Grammatical Error Correction: Tag, Not Rewrite',
-    long_description=open('README.md').read(),
-
+    version=__version__,
+    license="Apache-2.0 License",
+    description="GECToR – Grammatical Error Correction: Tag, Not Rewrite",
+    long_description_content_type="text/markdown",
```
Comment (desilinguist) [inline setup.py:23]: Oh wow! I didn't realize that markdown files now render correctly on PyPI. We should change all of the README.rst files to markdown in our other repos.

**Review by desilinguist — APPROVED** [review]
This looks great, @Frost45! It's much cleaner and, obviously, conda-friendly which is important for us!

File: setup.py [hunk 2]
```diff
@@ -1,30 +1,35 @@
...
-                 'Programming Language :: Python :: 3.8',
-                 ]
+    include_package_data=True,
+    classifiers=[
+        "Intended Audience :: Science/Research",
+        "Intended Audience :: Developers",
+        "Programming Language :: Python",
+        "Topic :: Scientific/Engineering",
+        "Operating System :: Unix",
+        "Programming Language :: Python :: 3.8",
```
Comment (ksteimel) [inline setup.py:33]: can we include multiple python versions in the classifier here?

**Review by ksteimel — APPROVED** [review]

File: train.py [hunk 3]
```diff
@@ -13,7 +13,7 @@
 from gector.seq2labels_model import Seq2Labels
```
[the remark this answers is not in the document]
Comment (Frost45) [reply train.py:13]: Nope, we haven't updated anything here except for the import. Everything else is autoformatting.

**Pull request merged**